packages/lesswrong/components/sunshineDashboard/SunshineSidebar.tsx

```diff
@@ -79,34 +78,11 @@ const SunshineSidebar = ({classes}: {classes: ClassesType<typeof styles>}) => {
           <AFSuggestCommentsList />
           <AFSuggestUsersList />
         </div>}
+        <SunshineCuratedSuggestionsList terms={{view:"sunshineCuratedSuggestions", limit: 7}} atBottom/>
       </div>}
 
-      {userCanDo(currentUser, 'posts.moderate.all') && <div>
-        { showSidebar ? <div className={classes.toggle} onClick={() => setShowSidebar(false)}>
-          Hide Full Sidebar
-            <KeyboardArrowDownIcon />
-          </div>
-          :
-          <div className={classes.toggle} onClick={() => setShowSidebar(true)}>
-            Show Full Sidebar
-            <KeyboardArrowRightIcon />
-          </div>}
-      </div>}
-
-
-      { showSidebar && userCanDo(currentUser, 'posts.moderate.all') && <div>
-        {!!currentUser!.viewUnreviewedComments && <SunshineNewCommentsList terms={{view:"sunshineNewCommentsList"}}/>}        
-        <SunshineCuratedSuggestionsList terms={{view:"sunshineCuratedSuggestions", limit: 50}} belowFold/>
-
-        {/* regular admins (but not sunshines) see AF content below the fold */}
-        { userIsAdmin(currentUser) && <div>
-          <AFSuggestUsersList />
-          <AFSuggestPostsList />
-          <AFSuggestCommentsList />
-        </div>}
-      </div>}
-
-      { showSidebar && <div>
```

Comment:
I don't think you intended to get rid of most of this?

Reply:
I did! I wanted to get rid of the first fold. I move the newcommentslist out of the sidebar, and now non-AF-admins don't see AF suggestions. Given how little we (or at least I) use that stuff, it seemed worth it for the simplification

Reply:
I personally don't use the sidebar for moderation so idc, but @Raemon or @darkruby501 might object to the removal of new comments from it.